clutch/tests/clutch-smoke-test.rs

```diff
@@ -1,14 +1,41 @@
 use assert_cmd::prelude::*;
+use std::path::{Path, PathBuf};
 use std::process::Command;
 
 fn lurkrs_cmd() -> std::process::Command {
     Command::cargo_bin("clutch").unwrap()
 }
 
+fn demo_file(name: &str) -> PathBuf {
+    let demo_dir = Path::new("demo/");
```

Comment:
Would it be less fragile to make this relative to `CARGO_MANIFEST_DIR`?

Reply:
Done.